automatic/wps-office-free/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,22 @@
+
+  $version = '10.2.0.5820'
+  $regName = $packageName -replace('\-',' ')
```

Comment:
@gep13 I agree. I don't think it should be relied on either, especially since they are undocumented.
It's not a problem anymore in this PR though, since it have been changed.